src/components/ui/Button.astro

```diff
@@ -1,9 +1,30 @@
 ---
-import SHA from './SHA.astro';
+import { cva, type VariantProps } from 'class-variance-authority';
 
-interface Props {
-  color?: 'orange' | 'green' | 'lightblue' | 'semidarkblue';
-  size?: 'sm' | 'lg' | 'lg-full' | 'sm-mobfull';
+import SHA from '@/components/ui/SHA.astro';
+import Link from '@/components/util/Link.astro';
+
+export const buttonVariants = cva(
+  'font-bold text-white drop-shadow transition-colors rounded-full text-center',
```

Comment:
This is an interesting design change, though I'm not really a fan of it, this comes especially into play when seeing how buttons behave inside cards as seen on the learn site. The fully rounded corners do not really seem to fit into the more boxy card shape, while the old design aligns more with its parent design and at least in my opinion looks more coherent.